
float64 pos_x
float64 pos_y
float64 pos_z

float64 r
float64 p
float64 y

bool collision_flag

